docs/cloud/pouta/vm-flavors-and-billing.md

```diff
@@ -205,6 +209,15 @@ flavors have faster CPUs and no overcommitment of CPU cores.
 
 **cPouta HPC flavor characteristics:**
 
+**hpc.5.\*:**
+
+-   Redundant power
+-      CPU:   AMD EPYC 7702 64-Core Processor,
```

Comment:
Ah, missed this one. This is getting rendered strangely in github, could you fix this and remove the tailing "," as well. Can also fix the indent for hpc.4 at the same time.

Reply:
The issue is fixed